### DISCUSSION TITLE
Where / how do I collect the values from selected table rows to be sent to a REST API?

### DISCUSSION THREAD

**Discussion by eugene-goldberg** [ToolJet/ToolJet#3995] [Q&A]
I have a table with 3 columns, of which the first one is of type "toggle switch". As the user selects / toggles multiple rows, I need a way to collect the selected row values, add them to a json array for the REST API call at some later point in time. How do I accomplish this?

**Comment by shivangT** [comment]
Hey @eugene-goldberg 
Let me reproduce this issue and get back to you to resolve this. 
Thanks

**Comment by shivangT** [comment]
Hey @eugene-goldberg 

Assuming your toggle switch field is made 'editable', 
Any change in the toggle switch will be exposed to `dataUpdates`  which you can find in the inspector on the left under `components  > table1 > dataUpdates`. Refer to the screenshot below.

- **No entry / 0item in the `dataUpdates`**

<img width="1440" alt="Screenshot 2022-09-15 at 6 49 06 PM" src="https://user-images.githubusercontent.com/32157641/190414059-92da24db-b858-4b7f-93a9-1d800a6ac6a9.png">


- **Clicking on 3 Toggle switches updates 3 new entries in `dataUpdates`**

<img width="1440" alt="Screenshot 2022-09-15 at 6 51 05 PM" src="https://user-images.githubusercontent.com/32157641/190414565-9c394e16-7dd3-42c5-816f-ef7f057521f9.png">

**Note:** You can access this data (`dataUpdates`) to create an array required for your rest API.
